### PR TITLE
Made intermediate login page

### PR DESCRIPTION
**Pull request by R303Vermeulen**
#55 
Changed initial login page to title page
Made Intermediate login page
Added extra style to textbox component

### CODE REVIEW

**Comment by R303Vermeulen** [comment]
IMG_7630

File: Eat_and_Greet/src/screens/Temp/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { View } from 'react-native';
+import { ScrollView, View } from 'react-native';
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/screens/Temp/index.tsx:1]: Importing ScrollView But I don't see it used

**Review by MorrowCameron — APPROVED** [review]
Allowing the unused ScrollView import assuming it will be added later

**Comment by MorrowCameron** [comment]
Pulled locally, it could use a scroll view for smaller screens but it still functions.

**Pull request merged**